flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["aarch64", "x86_64"]
```

Comment:
these are the only arches, this file isn't needed.

But then you extract a 'binary' so this is probably incorrect.